src/xdist/plugin.py

```diff
@@ -101,15 +101,6 @@ def pytest_addoption(parser: pytest.Parser) -> None:
         "--dist",
         metavar="distmode",
         action="store",
-        choices=[
```

Comment:
Based on expected usage  why not introduce a entrypoint 

That way we get choices and extension